pkg/apis/configuration/validation/validation.go

```diff
@@ -801,6 +801,10 @@ func validateRedirectURL(redirectURL string, fieldPath *field.Path, validVars ma
 		return append(allErrs, field.Required(fieldPath, "must specify a url"))
 	}
 
+	if !strings.Contains(redirectURL, "://") {
```

Comment:
could you also update the maps of supported variables?

For the error page returns, both headers and body, we need to support `$upstream_status` instead of `$status`

For the error page redirect, we do not need to support the `$status` anymore.

Reply:
Do we still support `upstream_status` in the error page redirect tho?